week-4/E-arrays-of-objects/exercise-3.js

```diff
@@ -55,21 +55,34 @@ WRITE YOUR CODE BELOW
 */
 
 
-var restaurantFinderApplication = {
+var restaurantFinderApplication = {            
     applicationName: "Restaurant Finder",
     applicationVersion: "1.0",
     restaurants: restaurants,
-    findAvailableRestaurants: function(numberOfPeople) {
+    findAvailableRestaurants: function(numberOfPeople) {return restaurants
+      .filter(
+        restaurant =>
+          restaurant.totalSeats - restaurant.numberOfCustomers >= numberOfPeople
+      )
+      .map(function (restaurant){return restaurant.applicationName});
```

Comment:
There's a bug here. Are you sure the `restaurant` object has an `applicationName` property?